modules/web-console/docker/console/frontend/Dockerfile

```diff
@@ -21,4 +21,10 @@ COPY ./ignite-web-console-*/frontend /data/www
 COPY ./frontend/nginx/nginx.conf /etc/nginx/nginx.conf
 COPY ./frontend/nginx/web-console.conf /etc/nginx/web-console.conf
 
-EXPOSE 80
+RUN mkdir -p /var/cache/nginx && \
```

Comment:
Why not https://hub.docker.com/r/twalter/openshift-nginx/dockerfile ?

Reply:
Let's follow OpenShift's documentation and examples for Dockerfiles https://docs.openshift.com/container-platform/3.11/creating_images/guidelines.html